app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -350,6 +355,9 @@ private void maybeRemoveUserPicture() {
     }
 
     private void updateUserPicture() {
+        if (!getGoogleApiClient().hasConnectedApi(Plus.API)) {
```

Comment:
Indeed, and how do I know that his "guard" is enough for the api to not complain about it. To much not-knowledge from my side.